pandas/tests/generic/test_generic.py

```diff
@@ -1016,6 +1016,30 @@ def test_pipe_panel(self):
             with pytest.raises(ValueError):
                 wp.pipe((f, 'y'), x=1, y=1)
 
+    def test_interpolate_axis(self):
```

Comment:
instead of adding a test, you can just paramaterize another one with axis_frame, which resolves to these pairs.

Reply:
@jreback I'm not really sure in which function you're asking me to resolve this.

The problem is that I couldn't find any proper function in [pandas/tests/generic/test_generic.py](https://github.com/pandas-dev/pandas/blob/1d1b14c7ea7c3e497b66453cacbb8230b26cb6a6/pandas/tests/generic/test_generic.py) to check interpolate function.

But I found proper methods under these classes in DataFrame tests and Series tests checking for interpolate.
https://github.com/pandas-dev/pandas/blob/1d1b14c7ea7c3e497b66453cacbb8230b26cb6a6/pandas/tests/frame/test_missing.py#L635
https://github.com/pandas-dev/pandas/blob/1d1b14c7ea7c3e497b66453cacbb8230b26cb6a6/pandas/tests/series/test_missing.py#L857

Should I resolve this in a function in [generic.py](https://github.com/pandas-dev/pandas/blob/1d1b14c7ea7c3e497b66453cacbb8230b26cb6a6/pandas/tests/generic/test_generic.py) or shifting my test function in one of the two classes I mentioned above?

Reply:
you can just add the axis fixture to one or more tests there (don't go overboard)